rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,65 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+# the lines below are sed'ed by build-server-rpm script to set a correct version
+# see: https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+%global commit            0000000000000000000000000000000000000000
+%define full_pmm_version 2.0.0
+
+%global shortcommit       %(c=%{commit}; echo ${c:0:7})
+%define build_timestamp   %(date -u +"%y%m%d%H%M")
+%define release           2
+%define rpm_release       %{release}.%{build_timestamp}.%{shortcommit}%{?dist}
+
+%global install_golang    0
+
+Name:       dbaas-controller
+# the line below is sed'ed by build-server-rpm script to set a correct version
+# see: https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+Version:    %{version}
+Release:    %{rpm_release}
+Summary:    Simplified API for managing Percona Kubernetes Operators
+
+License:    AGPLv3
+URL:        https://github.com/percona-platform/dbaas-controller
+Source0:    https://github.com/percona-platform/dbaas-controller/archive/%{commit}/dbaas-controller-%{shortcommit}.tar.gz
+
+%if %{install_golang}
+BuildRequires:   golang >= 1.14.0
+%endif
+
+%description
+dbaas-controller exposes a simplified API for managing Percona Kubernetes Operators
+See the PMM docs for more information.
+
+
+%prep
+%setup -q -n dbaas-controller-%{commit}
+
+
+%build
+export PMM_RELEASE_VERSION=%{full_pmm_version}
+export PMM_RELEASE_FULLCOMMIT=%{commit}
+export PMM_RELEASE_BRANCH=""
+
+# cd src/github.com/percona-platform/dbaas-controller
+make release
+
+%install
+install -d -p %{buildroot}%{_bindir}
+install -d -p %{buildroot}%{_sbindir}
+install -p -m 0755 bin/dbaas-controller %{buildroot}%{_sbindir}/dbaas-controller
+
+
+%files
+%license LICENSE
+%doc README.md
+%{_sbindir}/dbaas-controller
+
+
+%changelog
+* Fri Jul  10 2020 Mykyta Solomko <mykyta.solomko@percona.com> - 2.9.0-1
```

Comment:
seems version should be updated

Reply:
Done. Incremented build version

Reply:
I don't think @EvgeniyPatlan's comment was about _build_ version – we are going to release 2.9.1 today, and that package will not be installed until _at least_ 2.10, so 2.9.0 is outdated for sure

Reply:
(not to mention date in the changelog)

Reply:
Date and version has been updated